app/routes/notifications/get.js

```diff
@@ -0,0 +1,22 @@
+import { notificationCollection } from '@root/database'
+import { format } from 'date-fns/fp'
+
+export default async (req, res) => {
+    try {
+        const results = await notificationCollection().where('status', '==', false).get()
```

Comment:
```suggestion
        const notifications = await notificationCollection().where('status', '==', false).get()
```
Do result là 1 cái tên khá vô nghĩa, hãy đặt những cái tên nó rõ nghĩa hơn để sau này đọc lại còn hiểu được.

Reply:
à ok, nhìn xuống bên dưới thì đoạn này chấp nhận được. Sr em nha.